x/airdrop/keeper/msg_server_test.go

```diff
@@ -0,0 +1,24 @@
+package keeper_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+    keepertest "github.com/Stride-Labs/stride/v22/testutil/keeper"
+    "github.com/Stride-Labs/stride/v22/x/airdrop/types"
+    "github.com/Stride-Labs/stride/v22/x/airdrop/keeper"
+)
+
+func setupMsgServer(t testing.TB) (keeper.Keeper, types.MsgServer, context.Context) {
```

Comment:
would prefer to use the testing framework we use in the other modules but open to discuss it if you have a strong preference!